test/dotnet-new3.UnitTests/Approvals/PostActionTests.RunScript_Basic_Approval.OSX.verified.txt

```diff
@@ -0,0 +1,7 @@
+﻿The template "TestAssets.PostActions.RunScript.Basic" was created successfully.
+
+Processing post-creation actions...
+
```

Comment:
Need to investigate why empty line is appearing on Unix system here. Not related to this PR, may be done in separate issue if meets the bar.